3.10/foxx-reference-modules-queues.md

```diff
@@ -370,7 +370,7 @@ assertEqual(queue.complete(logScript).length, 1);
 
 Returns an array of job ids of jobs in the given queue with the status
 `"progress"`, optionally filtered by the given job type.
-The jobs will be looked up in the specified queue in the current database.
+The jobs is looked up in the specified queue in the current database.
```

Comment:
```suggestion
The job is looked up in the specified queue in the current database.
```

Reply:
I think I didn't read this carefully enough and it should be `The jobs are looked up`